process/datadog_checks/process/process.py

```diff
@@ -214,7 +207,7 @@ def psutil_wrapper(self, process, method, accessors, try_sudo, *args, **kwargs):
             if method == 'num_fds' and Platform.is_unix() and try_sudo:
                 try:
                     # It is up the agent's packager to grant corresponding sudo policy on unix platforms
-                    result = int(subprocess.check_output(['sudo', sys.executable, __file__, method, str(process.pid)]))
+                    result = int(subprocess.check_output('sudo ls /proc/{}/fd/ | wc -l'.format(process.pid), shell=True))
```

Comment:
Is `shell=True` necessary?

Reply:
I agree, we should avoid `shell=True` and do the split count on our own.